Reconstruction/RecCalorimeter/tests/options/runFullCaloSystem_SimAndFullDigitisation.py

```diff
@@ -0,0 +1,197 @@
+import os
+import numpy as np
+
+# simulations setup                                                                                                                                                                                                              
+energy=1000000
+num_events=1
```

Comment:
for the events, it can be in fact given as argument
if args.nevents is not None:
    num_events = args.nevents
and then when you call the script, you add the option --nevents=1234, if nothing, the default is used

Reply:
Hi Clement, thanks for the suggestion! But could you please be more specific? Do i need to import the GaudiKernel.ProcessJobOptions ? Or something else?
Thanks,
Coralie

Reply:
sorry for not mentioning it before, yes, you need to import something before, so the full syntax would be:

from FWCore.joboptions import parse_standard_job_options
args = parse_standard_job_options()
num_events=100
if args.nevents is not None:
    num_events = args.nevents

Reply:
Thank you, I added it, but it only gives me an error.. it's complaining that gaudirun.py (which I call to run my script) does not know the option --nevents..

Reply:
ah yes, it's because you need to use fccrun
#236 was supposed to remove all references to gaudirun, but was not approved yet